packages/yarnpkg-core/sources/Project.ts

```diff
@@ -1276,6 +1378,9 @@ export class Project {
     if (mode === InstallMode.SkipBuild)
       return;
 
+    for (const [, explain] of miscUtils.sortMap(skippedBuildLogs, ([locator]) => structUtils.stringifyLocator(locator)))
```

Comment:
That doesn't work, since `skippedBuildLogs` is a `Array<[U, V]>` - `values()` still return a `[U, V]` (no changes, since the default iteration on arrays does the same).

Reply:
Ah, my bad, thought it was a `Map`, you're right.